### ISSUE TITLE
请问这个包只能处理10x的数据吗？现成的矩阵是否可以呢？

### ISSUE THREAD

**Issue by Yiquan-Jiang** [wguo-research/scCancer#16]
比如这个数据已经经过了如下的处理，可以用您们的包直接读入吗？
Single-cell suspensions were converted to barcoded scRNA-seq libraries by using the Chromium Single Cell 3’ Library, Gel Bead & Multiplex Kit and Chip Kit (10x Genomics), aiming for an estimated 5,000 cells per library and following the manufacturer’s instructions. Samples were processed using kits pertaining to V2 barcoding chemistry of 10x Genomics. Single samples are always processed in a single well of a PCR plate, allowing all cells from a sample to be treated with the same master mix and in the same reaction vessel. For each patient, all samples (NTL, PT, PVTT and MLN) were processed in parallel in the same thermal cycler. The generated scRNA-seq libraries were sequenced on an Illumina NovaSeq sequencer. The Cell Ranger software (version 2.2.0) was used to perform sample demultiplexing, barcode processing and single-cell 3’ counting. Cell Ranger’s mkfastq function was used to demultiplex raw base call files from the sequencer, into sample-specific fastq files. Afterward, fastq files for each sample were processed with Cell Ranger’s count function, which was used to align reads to human genome (build hg38) and quantify gene expression levels in single cells.
该数据直接是一个normalized之后的txt文件的矩阵

**Comment by wellgoo** [comment]
Yes, it can ship the first cell calling visualization step and use all the following functions. You need to read the detailed manual. We will provide a modified function for processing the pre-processed matrix data soon.

**Comment by MedicalKong** [comment]
> Yes, you can skip the first cell calling visualization step and use all the following functions. Please read the detailed manual about how to use them.
> 
> We will provide a modified function for processing the pre-processed matrix data soon.

I am sorry to bother you again. 
I could'n find where the detailed manual is. Did you refer to the wiki page?
I didn't see detailed methods about how to skip the cell calling step. 
Did you mean to skip the "cell calling" step by modifying parts of the code in running scStatistics as follows?

stat.results <- runScStatistics(
  dataPath = dataPath,
  savePath = savePath,
  sampleName = sampleName,
  authorName = authorName,
????
)

**Comment by wguo-research** [comment]
You can use `help(package="scCancer")` to view the introduction of all the functions in `scCancer`, and use these single-step functions to analyze your data. More convenient ways to start with a data matrix will be developed soon.

**Comment by MedicalKong** [comment]
> You can use `help(package="scCancer")` to view the introduction of all the functions in `scCancer`, and use these single-step functions to analyze your data. More convenient ways to start with a data matrix will be developed soon.

Thank you very much. Your package is a good job!

**Comment by MedicalKong** [comment]
> You can use `help(package="scCancer")` to view the introduction of all the functions in `scCancer`, and use these single-step functions to analyze your data. More convenient ways to start with a data matrix will be developed soon.

Dear author of "scCancer", you said that"More convenient ways to start with a data matrix will be developed soon.". Could you please tell me how soon  this function will be realized?
